compose-oneliner.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+set -x
```

Comment:
why do we need this ?

Reply:
we don't it was left out on the last commit.
i've committed all the above for my on testing the last commit is the one with all the changes.